parcels/tools/_helpers.py

```diff
@@ -56,3 +59,13 @@ def deprecated_made_private(func: Callable) -> Callable:
 
 def patch_docstring(obj: Callable, extra: str) -> None:
     obj.__doc__ = f"{obj.__doc__ or ''}{extra}".strip()
+
+
+@np.vectorize
```

Comment:
Is it really necessary to vectorise this function? Is it really so compute-intense? 
Or is this a start of more of these decorators?